<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formProfileManager</class>
 <widget class="QMainWindow" name="formProfileManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>416</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Profile Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>360</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
    <property name="icon">
     <iconset>
      <selectedon>../../images/24x24/add_24x24.png</selectedon>
     </iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>360</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
    <property name="icon">
     <iconset>
      <selectedon>../../images/edit.png</selectedon>
     </iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>360</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
    <property name="icon">
     <iconset>
      <selectedon>../../images/24x24/trash_24x24.png</selectedon>
     </iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDeleteAll">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>360</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete All</string>
    </property>
    <property name="icon">
     <iconset>
      <selectedon>../../images/24x24/trash-circle-red-5121.png</selectedon>
     </iconset>
    </property>
   </widget>
   <widget class="QTableWidget" name="tbProfile">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>791</width>
      <height>341</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>150</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Profile Name</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>tbProfile</sender>
   <signal>itemDoubleClicked(QTableWidgetItem*)</signal>
   <receiver>btnEdit</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
